MainScreenStackWidget.py

```diff
@@ -0,0 +1,44 @@
+import sys
+from PySide6.QtWidgets import (
+    QApplication,
+    QMainWindow,
+    QVBoxLayout,
+    QPushButton,
+    QStackedWidget,
+    QLabel,
+)
+
+from StartMenuTest import (
+    StartMenu,
+)  # Assuming StartMenuTest.py contains StartMenu class
+from MapScreen import MapScreen  # Assuming MapScreen.py contains MapScreen class
+
+
+class MainWindow(QMainWindow):
```

Comment:
Perhaps replace `main.py` with this file. We only want one window to be created.